documentation/src/main/assembly/dist.xml

```diff
@@ -72,13 +95,61 @@
-            <directory>${asciidoctor.base-output-dir}/migration/</directory>
-            <outputDirectory>migration/</outputDirectory>
-            <includes>
-                <include>*/*.html</include>
-                <include>*/*.pdf</include>
-            </includes>
-        </fileSet>
     </fileSets>
+
+    <files>
+        <!-- HTMLs: -->
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/public/reference/index.html</source>
+            <outputDirectory>reference/en-US/html_single/</outputDirectory>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/public/getting-started/orm/index.html</source>
+            <outputDirectory>getting-started/orm/en-US/html_single/</outputDirectory>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/public/getting-started/standalone/index.html</source>
+            <outputDirectory>getting-started/standalone/en-US/html_single/</outputDirectory>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/public/getting-started/index.html</source>
+            <outputDirectory>getting-started/en-US/html_single/</outputDirectory>
+        </file>
+
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/migration/index.html</source>
+            <outputDirectory>migration/html_single/</outputDirectory>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/html/internals/index.html</source>
+            <outputDirectory>internals/html_single/</outputDirectory>
+        </file>
+        <!-- PDFs: -->
+        <file>
+            <source>${asciidoctor.base-output-dir}/pdf/public/reference/index.pdf</source>
+            <outputDirectory>reference/en-US/pdf/</outputDirectory>
+            <destName>hibernate-search-reference.pdf</destName>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/pdf/public/getting-started/orm/index.pdf</source>
+            <outputDirectory>getting-started/orm/en-US/pdf/</outputDirectory>
+            <destName>hibernate-search-getting-started-orm.pdf</destName>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/pdf/public/getting-started/standalone/index.pdf</source>
+            <outputDirectory>getting-started/standalone/en-US/pdf/</outputDirectory>
+            <destName>hibernate-search-getting-started-standalone.pdf</destName>
+        </file>
+
+        <file>
+            <source>${asciidoctor.base-output-dir}/pdf/migration/index.pdf</source>
+            <outputDirectory>migration/pdf/</outputDirectory>
+            <destName>hibernate-search-migration.pdf</destName>
+        </file>
+        <file>
+            <source>${asciidoctor.base-output-dir}/pdf/internals/index.pdf</source>
+            <outputDirectory>internals/pdf/</outputDirectory>
+            <destName>hibernate-search-internals.pdf</destName>
+        </file>
```

Comment:
This needs to be conditional, or to ignore missing files, because PDFs are not generated by default (it's too slow).

I'm getting this when I don't generate PDFs:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.4.2:single (make-dist) on project hibernate-search-documentation: Failed to create assembly: Error adding file to archive: /home/yrodiere/workspaces/main/hibernate-search/documentation/target/asciidoctor/pdf/public/reference/index.pdf -> [Help 1]
```

Reply:
ohh right, good point 😃 will do something about it 👍🏻

Reply:
I didn't see anything useful in maven-assembly-plugin to make pdfs optional, so I've moved the copying of pdfs to a different descriptor file, which is included in a pdf profile.